projects/metis/src/app/dataset/executions-data-grid/executions-data-grid.component.ts

```diff
@@ -40,13 +45,13 @@ export class ExecutionsDataGridComponent {
   /** goToPreview
   /* fire the preview event
   */
-  goToPreview(execution: WorkflowExecution, pluginExecution: PluginExecution): void {
+  goToPreview(executionId: string, pluginExecution: PluginExecution): void {
     const previewFilters: PreviewFilters = {
       baseFilter: {
-        executionId: execution.id,
+        executionId: executionId,
         pluginType: pluginExecution.pluginType
       },
-      baseStartedDate: execution.startedDate
+      baseStartedDate: pluginExecution.startedDate
```

Comment:
Are you sure this change is correct? The starting date of the workflow execution is not the same as the starting date of the plugin.